app/assets/javascripts/admin/addon/models/admin-plugin.js

```diff
@@ -24,6 +24,10 @@ export default class AdminPlugin {
     return this.name.replaceAll("-", "_");
   }
 
+  get kebabCaseName() {
+    return this.name.replaceAll(" ", "-").replaceAll("_", "-");
```

Comment:
i wonder if we dont have a helper for this

Reply:
Not from what I can see, but also we have a shitload of helpers so maybe I am missing it / it's a weird name?

Reply:
@martin-brennan I think you're looking for `import { dasherize } from "@ember/string";`